docs/public-networks/how-to/connect/configure-ports.md

```diff
@@ -9,7 +9,7 @@ tags:
 
 # Configure ports
 
-To enable communication you must expose Hyperledger Besu ports appropriately. The following shows an example port configuration for a Besu node on AWS.
+To enable communication you must expose Hyperledger Besu ports appropriately. If your nodes are running in AWS, ensure you have appropriate `SecurityGroups` to allow access to the required ports. The following shows an example port configuration for a Besu node on AWS. 
```

Comment:
I would add the "If your nodes are running in AWS..." sentence as a **Note** just above the tip in this section.